kvmhelpers/GUESTBRIDGE-down.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+sh ./GUESTBRIDGE-vars.sh
```

Comment:
Probably need to do something around `dirname "$0"` instead of `.` to support invoking these scripts from other directories,, but that can be FUTUREWORK.